dist/bu-blocks-frontend.js

```diff
@@ -758,7 +788,7 @@ const bu_blocks = {};
 	 * Find all Collapsible blocks
 	 */
 	var findElements = function() {
```

Comment:
This also looks very similar to code above - same q about the `wp-block` prefix here

Reply:
same as before, this function exists in each of the 2 collapsible blocks